chipy_org/libs/social_auth_pipelines.py

```diff
@@ -18,6 +18,4 @@ def associate_by_email(*args, **kwargs):
         if User.objects.filter(email=email).exists():
             msg = ugettext('This email is already in use. First login with your other account and '
                            'under the top right menu click add account.')
-            raise AuthAlreadyAssociated(backend, msg % {
-                'provider': backend.name
-            })
+            raise AuthAlreadyAssociated(backend, msg.format(provider=backend.name))
```

Comment:
I don't see either one of these formatted strings doing anything besides return the string in the ugettext argument.   Maybe the backend was supposed to be provided.  Maybe we could add it now?  

Also, why do we use `ugettext` here? Isn't that for translation? Do we do translation for chipy.org?

Reply:
In the code's current format, the string representation of the 'backend' object will show up in the raise error message since 'backend' is in the parentheses.  This 'backend' object string will be concatenated with the variable 'msg' since there's a comma between them.

A raise statement in Python is usually in the form of:
 `raise SomeError("some error string")`

What will not show up in our raised error message is 'backend.name' specifically. This is because 'msg' does not have 'backend.name' in it.

So in line with @elmq0022's  suggestions, maybe the solution might look like this:

```
msg = f"Backend is {backend}. Backend name is {backend.name}. This email from is already in use. First login with your other account and under the top right menu click add account."

raise AuthAlreadyAssociated(msg)
```

@raymondberg Feel free to change it however you see fit.